tests/lib/__init__.py

```diff
@@ -1082,6 +1094,9 @@ def create_basic_wheel_for_package(
     requires_python=None,
     extra_files=None,
 ):
+
+    # these can't be made default args because some tests/functions pass None as an arg.
+    # for e.g create_basic_wheel_for_package("scriptname", "name", "v1.0.0", None)
```

Comment:
This context is not necessary, and is incorrect. See https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html for details on mutable default arguments.

Reply:
I did not know about this.